File: strategy/def-bank-remit.ts
import { Fee } from "./fee/fee";
import { Remit } from "./remit";

export class DefBankRemit extends Remit {
    constructor(fee: Fee) {
        super(fee)
    }

    public createTransferMessage(): string {
        return "Def Bank! I will send the money! give me result!";
    }
}

File: src/main/java/strategy/DefBankRemit.java
package strategy;

import strategy.fee.Fee;

public class DefBankRemit extends Remit {
    public DefBankRemit(Fee fee) {
        super(fee);
    }

    @Override
    public String createTransferMessage() {
        return "Def Bank! I will send the money! give me result!";
    }
}
